src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -42,6 +42,7 @@ object ApiOperations {
 class ApiOperations extends DataOperations with MetadataOperations {
   import ApiOperations._
   lazy val dbReadHandle: Database = DatabaseUtil.conseilDb
+  private val MAX_NOT_FOUND = -1
```

Comment:
Just to be sure, there should be another constant like this already defined, but it might be in another file, so probably redefining the same here might help keep things properly decoupled...

Reply:
oh, yes it is in `TezosDatabaseOperations`